designer/server/src/main/scala/pl/touk/nussknacker/ui/db/entity/CommentEntityFactory.scala

```diff
@@ -46,6 +48,7 @@ final case class CommentEntityData(
     processVersionId: VersionId,
     content: String,
     user: String,
+    impersonatedBy: Option[String],
```

Comment:
I see some inconsistency here.
- in the `user` field we have a name of a user

but:
- in the `impersonatedBy` we have an ID of a user
- there is no foreign key in the schema definition
- the name of the fields doesn't suggest it's an ID

Reply:
Hmmm, the situation seems to be a bit more complex.
User's id is not an ID in a database sense because AFAIK there is no user table with information about users. Instead id is an identity that is passed from `AuthenticationResources` implementation to `AuthenticatedUser` an then to the `LoggedUser`. 

In Nu there seems to be a confusion between user's name and `identity`. It is `username` that is saved under the `user` field although it should probably be also the identity however it would probably introduce difficulties with displaying the user's name in the designer.

Initially `impersonatedBy` was representing impersonating user's name. However, I changed it to represent identity after Arek's comment. @arkadius  WDYT?

Reply:
After offline consultations two columns regaring impersonation will be added. First one holding the Id and second the name of an impersonating user. 
As there are some plans for switching from using username in favor of identity in Nu db tables this will let us simply delete the additional `impersonatedByUsername` column once it's not needed. For now I would add it so it's possible to present impersonating user's name on the frontend.